scripts/build-merge

```diff
@@ -127,6 +127,11 @@ echo-do cp examples/movies/spark/target/scala-2.10/movies-spark.jar ${STAGE_MOVI
 echo-do mkdir -p stage/docs/scala/store
 echo-do mv doc/target/scala-2.11/unidoc stage/docs/scala/store/${version}
 
+#set NSEEDS
```

Comment:
Hi Topher,

With regards to the below change, I also had to set NSEEDS in two other
scripts namely
-build-pull-request ( what comment do I add here?)
-build-periodic-test ( would number of seeds for periodic-test suffice for
this?)


I had one more doubt regarding removing intensity from
*disk/stub/com/treode/disk/stubs/CrashChecks.scala.*

There are actually more uses for ncrashes in this file, in the function
forAllCrashes(seed : Long). What do I do about these?

Thank you.

With Regards,
Sanjay

On 20 January 2015 at 20:32, Topher <notifications@github.com> wrote:

> In scripts/build-merge
> <https://github.com/Treode/store/pull/95#discussion-diff-23227092>:
>
> > @@ -127,6 +127,11 @@ echo-do cp examples/movies/spark/target/scala-2.10/movies-spark.jar ${STAGE_MOVI
> >  echo-do mkdir -p stage/docs/scala/store
> >  echo-do mv doc/target/scala-2.11/unidoc stage/docs/scala/store/${version}
> >
> > +#set NSEEDS
>
> This comment tells me what the line of code tells me. Maybe change the
> comment to:
> Number of seeds for psuedo-random tests.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Treode/store/pull/95/files#r23227092>.
>